Objects/DiscoverMovie.swift

```diff
@@ -0,0 +1,19 @@
+
+import Foundation
+
+final class DiscoverMovie: Codable {
+    var page: Int?
+    var results: [Movie] = []
```

Comment:
In this class, you have the initialization, so you don't need to set the value directly. Delete " = [] "